### PR TITLE
Data model basics

### LINKED ISSUE #11: Define initial app and data model
Initial ideas:
- `Project`
  - collaborators (m2m with users, with a through model for their role on the project)
- `DataInterface` (inputs)
  - database connection string, etc.
- `Pipeline`
  - name
  - collection of step
- `Step`
  - `type`
  - `script` - `TextField` editable by monaco

Include django admin for editing all models

### PR DESCRIPTION
**Pull request by wfzyx**
Adds the primordial models for the project

How to test:
- python manage.py makemigrations
- python manage.py migrate

Closes #11

### CODE REVIEW

File: .envs/.local/.django [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # General
 # ------------------------------------------------------------------------------
-USE_DOCKER=yes
+USE_DOCKER=no
```
Comment (tobiasmcnulty) [inline .envs/.local/.django]:
```suggestion
USE_DOCKER=yes
```

**Review by ronardcaktus — APPROVED** [review]
LGTM

File: config/settings/base.py [hunk 2]
```diff
@@ -42,6 +42,7 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#databases
 DATABASES = {"default": env.db("DATABASE_URL")}
 DATABASES["default"]["ATOMIC_REQUESTS"] = True
+
```
Comment (tobiasmcnulty) [inline config/settings/base.py]:
```suggestion
```

File: config/settings/base.py [hunk 3]
```diff
@@ -70,12 +71,13 @@
     "crispy_bootstrap5",
     "allauth",
     "allauth.account",
-    "allauth.socialaccount",
-    "django_celery_beat",
+    "allauth.socialaccount"
+    # "django_celery_beat",
```
Comment (tobiasmcnulty) [inline config/settings/base.py]:
```suggestion
    "allauth.socialaccount",
    "django_celery_beat",
```

File: config/settings/base.py [hunk 4]
```diff
@@ -251,29 +253,6 @@
     "root": {"level": "INFO", "handlers": ["console"]},
 }
 
-# Celery
```
Comment (tobiasmcnulty) [inline config/settings/base.py]: Please restore the celery config for now.

File: etlman/projects/admin.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        StepInline,
+    ]
+
+
+class StepAdmin(admin.ModelAdmin):
+    list_display = ["name", "step_order"]
+    list_filter = ("pipeline__project__name",)
+
+
+# Register your models here.
```
Comment (tobiasmcnulty) [inline etlman/projects/admin.py]:
```suggestion
```

File: requirements/base.txt [hunk 6]
```diff
@@ -6,7 +6,7 @@ argon2-cffi==21.3.0  # https://github.com/hynek/argon2_cffi
 whitenoise==6.2.0  # https://github.com/evansd/whitenoise
 redis==4.3.3  # https://github.com/redis/redis-py
 hiredis==2.0.0  # https://github.com/redis/hiredis-py
-celery==5.2.7  # pyup: < 6.0  # https://github.com/celery/celery
+# celery==5.2.7  # pyup: < 6.0  # https://github.com/celery/celery
```
Comment (tobiasmcnulty) [inline requirements/base.txt]:
```suggestion
celery==5.2.7  # pyup: < 6.0  # https://github.com/celery/celery
```

**Review by tobiasmcnulty — APPROVED** [review]
🎉 Looks good! Just a few nitpicks. Merge at will.

File: config/settings/base.py (hunk 4 above)
Comment (wfzyx) [reply config/settings/base.py]: Done !

**Pull request merged**